embed_temp/embed_temp.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+class Embed_Template:
```

Comment:
Please can you:
- Rename "Embed_Template" class to “EmbedTemplate”
- Move embed_template.py file be moved into a folder called static/embed_template.py
- Create an init file in static/__init__.py and add this class to it
- Add the static folder to the main __init__.py file
- Import discord in the EmbedTemplate class